### DISCUSSION TITLE
admin pannel

### DISCUSSION THREAD

**Discussion by drkamine** [umputun/remark42#1115] [Q&A]
hello , i'm fresher with remark42 i runing a docker latest and i want to access to admin 
but i don't get http:IP:8080/admin

**Comment by umputun** [comment]
I'm not sure why you expecting anything on /admin. Pls see the documentation showing admin tools for the privileged users

**Comment by drkamine** [comment]
thank you for your answer . i don't see any options linked to direct admin except in api . is there 's an option  to approuved comment from admin ??

**Reply by umputun** [reply]
see https://github.com/umputun/remark42/wiki/Admin-UI
